File: app/src/main/java/ru/sicampus/bootcamp2025/data/dto/UserDto.java
package ru.sicampus.bootcamp2025.data.dto;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class UserDto {
    @Nullable
    @SerializedName("id")
    public Integer id;
    @Nullable
    @SerializedName("firstName")
    public String firstName;
    @Nullable
    @SerializedName("lastName")
    public String lastName;
    @Nullable
    @SerializedName("username")
    public String username;
    @Nullable
    @SerializedName("password")
    public String password;
    @Nullable
    @SerializedName("contactInfo")
    public String contactInfo;
    @Nullable
    @SerializedName("biography")
    public String biography;
    @Nullable
    @SerializedName("photo")
    public String photo;
    @Nullable
    @SerializedName("role")
    public Integer role;
    @Nullable
    @SerializedName("status")
    public Integer status;
}

File: app/src/main/java/ru/sicampus/bootcamp2025/data/UserDTO.kt
package ru.sicampus.bootcamp2025.data

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable


@Serializable
data class UserDTO (
    @SerialName("name") //можно потом поменять если что
    val name: String,
    @SerialName("lastName")
    val lastName: String,
    @SerialName("age")
    val age: Int
)